src/server/src/Server.cpp

```diff
@@ -241,7 +280,6 @@ void Server::enable_force_control()
     drdEnableFilter(false);
 }
 
-
```

Comment:
The adopted convention is having two spaces between each function, can you please put it back? :)

Reply:
done